dotnet/test/common/BiDi/WebExtension/WebExtensionTest.cs

```diff
@@ -0,0 +1,76 @@
+// <copyright file="WebExtensionTest.cs" company="Selenium Committers">
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// </copyright>
+
+using NUnit.Framework;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace OpenQA.Selenium.BiDi.WebExtension;
+
+class WebExtensionTest : BiDiTestFixture
+{
+    [Test]
+    [IgnoreBrowser(Selenium.Browser.Chrome, "Web extensions are not supported yet?")]
```

Comment:
Try to support it according https://github.com/SeleniumHQ/selenium/pull/15850#issuecomment-2991670641